src/meshdb/urls.py

```diff
@@ -16,9 +16,22 @@
 """
 from django.contrib import admin
 from django.urls import include, path
+from drf_spectacular.views import SpectacularAPIView, SpectacularRedocView, SpectacularSwaggerView
+
+from meshapi.docs import SpectacularSwaggerInjectVarsView
 
 urlpatterns = [
+    path("", include("meshweb.urls")),
+    path("auth/", include("rest_framework.urls")),
     path("admin/", admin.site.urls),
     path("api/v1/", include("meshapi.urls")),
-    path("", include("meshweb.urls")),
+    path("api-docs/openapi3.json", SpectacularAPIView.as_view(), name="api-docs-schema"),
+    path(
+        "api-docs/swagger/",
+        SpectacularSwaggerInjectVarsView.as_view(
+            url_name="api-docs-schema"  # , template_name="drf_spectacular/swagger_ui.html"
+        ),
+        name="api-docs-swagger",
+    ),
+    path("api-docs/redoc/", SpectacularRedocView.as_view(url_name="api-docs-schema"), name="api-docs-redoc"),
```

Comment:
Can we change `api-docs` to `docs`?

Reply:
These are just API docs and they will only ever be API docs, we can't add other stuff to the page without some template pain (and this should probably live on the wiki anyway). I stand by `api-docs`

Reply:
Well, I wasn't thinking of adding "other stuff" to these docs, it's just that these are _the_ docs for MeshDB. I see where you're coming from though. IDRC.